package.py

```diff
@@ -0,0 +1,32 @@
+name = "Red9"
+
+version = "2.75.0"
+
+authors = [
+    "Mark Jackson",
+    "Jeremy Andriambolisoa",
+]
+
+description = \
+    """
+    Red9 offers cutting edge production services, rigging, facial, tool solutions,
+    bespoke workflows and support to the animation industry.
+    """
+
+
+requires = [
+    "python-3+",
+    "maya-2025",
```

Comment:
You should probably set the version of maya to 2025+ since all the future version will be with Python 3+